viz-lib/src/visualizations/chart/Editor/ColorsSettings.test.js

```diff
@@ -70,7 +70,7 @@ describe("Visualizations -> Chart -> Editor -> Colors Settings", () => {
         .simulate("click");
     });
 
-    test("Sets custom color scheme", async done => {
+    test("Sets custom color scheme", done => {
```

Comment:
Was this `async` important for something I'm missing? It was causing this test to fail removing it didn't cause any harm 🤷‍♂️ 
(Probably some babel setting missing for it to work, but it was way easier to just remove it)

Reply:
I think `async` is not needed here, so it's ok to remove. Return value is always ignored, and `done` is used for async tests